banderwagon/element.go

```diff
@@ -75,7 +75,7 @@ func (p Element) BytesUncompressed() [UncompressedSize]byte {
 }
 
 // BatchNormalize normalizes a slice of group elements.
-func BatchNormalize(elements []*Element) {
+func BatchNormalize(elements []*Element) error {
```

Comment:
I've decided to make both `Normalize` and `BatchNormalize` return an error if they find a point at infinity case, which must not happen in Banderwagon circumstances.

Indirectly, this avoids some allocations since we don't have to track the `zeroes` slice. 

More about this in tests.